charts/jenkins/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Use the following links to reference issues, PRs, and commits prior to v2.6.0.
 
 The change log until v1.5.7 was auto-generated based on git commits. Those entries include a reference to the git commit to be able to get more details.
 
+## 2.6.5
+
+Added IRSA support for jenkins backup CronJob
```

Comment:
This doesn’t seem to be anything to do with IRSA. Added ability to configure annotations for the backup pod?

Reply:
Updated changelog to "Added security context for jenkins backup CronJob and annotations to its serviceaccount."

Reply:
Hi @timja, please let me know if any other changelog is more relevant.